.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:10
```

Comment:
Why did you updated the version ?

Reply:
Django added minimum database version checks in https://github.com/django/django/commit/9ac3ef59f9538cfb520e3607af743532434d1755
Django main requires at least Postgres 11
https://github.com/django/django/blob/21d8ea4eb3e22d458515b5278c8cd3a15b069799/django/db/backends/postgresql/features.py#L9